ksp_plugin/pile_up.hpp

```diff
@@ -236,6 +241,12 @@ class PileUp {
   // Called in the destructor.
   std::function<void()> deletion_callback_;
 
+  // A PID used to smoothen the value of the apparent angular momentum obtained
+  // from KSP.
+  PID<Bivector<AngularMomentum, ApparentPileUp>,
+      /*horizon=*/25,
+      /*finite_difference_order=*/5> pid_;
```

Comment:
`apparent_angular_momentum_`, or `apparent_angular_momentum_controller_`; no need to speak Hungarian, but it would be useful to document what is being controlled.

Reply:
`apparent_angular_momentum_controller_`